__tests__/fixture/post.ts

```diff
@@ -4,6 +4,17 @@ import { Post, PostKeyword, PostTag } from '../../src/entity';
 const now = new Date();
 
 export const postsFixture: DeepPartial<Post>[] = [
+  {
```

Comment:
Usually not a good idea to add a new fixture here.
This is used everywhere so it will cause other tests to fail I think.

Reply:
Oh, right, I'll do this as a specific addition on that specific case only.